### DISCUSSION TITLE
Upgrading to v6.3.0 failed with `invalid argument for flag `--log-cluster-name' (expected bool)`

### DISCUSSION THREAD

**Discussion by skreddy6673** [concourse/concourse#5867] [Help & Support] [answered]
Setting `logClusterName:` to `Concourse-Dev` at https://github.com/concourse/concourse-chart/blob/e67b1ebdaeafbf628eefcc43beb6dc956c6744af/values.yaml#L312

failed the upgrade with ```error: invalid argument for flag `--log-cluster-name' (expected bool): strconv.ParseBool: parsing "Concourse_Dev": invalid syntax``` error message.

I had concourse with this value set for a really long time, even v6.0.0 worked with this value set as `logClusterName: Concourse-Dev` .

**Steps To Reproduce:**
Install v6.3.0 helm chart with https://github.com/concourse/concourse-chart/blob/e67b1ebdaeafbf628eefcc43beb6dc956c6744af/values.yaml#L312 value set to `Concourse-Dev` or some random name.

**Expected results:**
When installed web nodes run successfully.

**Actual results**
```error: invalid argument for flag `--log-cluster-name' (expected bool): strconv.ParseBool: parsing "Concourse_Dev": invalid syntax```

**Additional context**
*Triaging info*
Concourse version: v6.3.0
Helm Chart Version: v11.2.3
Did this used to work? yes in 6.0.0, 5.8.1, 5.7.1 and all older versions.

**Workaround**
Set the value to True or false and it will work.

**Comment by jamieklassen** [comment, marked as answer]
I don't think this actually "worked" in older versions, it just didn't _break_, as described in [this release note](https://github.com/concourse/concourse/releases/tag/v6.2.0#strict-env-vars). `--log-cluster-name` was always supposed to be a bool, as you can see as far back as the v5.7.1 source code https://github.com/concourse/concourse/blob/v5.7.1/atc/atccmd/command.go#L159.

It sounds like your thinking was that this value should allow you to specify _which_ cluster name should appear in the logs, but the idea was that you independently specify the cluster name with the [`--cluster-name` flag](https://github.com/concourse/concourse/blob/v5.7.1/atc/atccmd/command.go#L155) and then use `--log-cluster-name` to opt in to having _that_ value appear in the logs: #4387

**Reply by skreddy6673** [reply]
You are correct, i should described it as it `didn't break`.
